pkg/api/VERSIONING.md

```diff
@@ -13,7 +13,7 @@ commit in this repository, possibly unreleased.
 
 | Release | autoscaler-agent | VM informant |
 |---------|------------------|--------------|
-| _Current_ | v1.0 - v1.2 | v1.1 - v1.2 |
+| Forthcoming | **v1.0 - v1.3** | **v1.1 - v1.2** |
```

Comment:
```suggestion
| _Current_ | **v1.0 - v1.3** | **v1.1 - v1.2** |
```

In the past we've (read: I've) left this as "Current" even when it's different from the latest release. I'd just stick with that tbh.

---

Some mostly unrelated context: The release flow (build+push images, etc) is triggered by pushing a tag; I'll make a commit directly to main that updates these tables, tag it (with a bunch of info in the tag), and then push everything.